esp_encrypted_img/include/esp_encrypted_img.h

```diff
@@ -152,6 +152,16 @@ bool esp_encrypted_img_is_complete_data_received(esp_decrypt_handle_t ctx);
 */
 esp_err_t esp_encrypted_img_decrypt_abort(esp_decrypt_handle_t ctx);
 
+/**
+* @brief  return size of struct pre_enc_bin_header which used as header size in checking wheather complete image 
+          received or not when config HTTP partial download is enabled. This header size is useful in pre_encrypted_ota
+          while comparing image written length with actual length. Struct pre_enc_bin_header contain magic, inc_gcm, 
+          iv, bin_size, auth, extra_header.
+*
+* @return
+*    - uint16_t header_size
```

Comment:
```suggestion
* @brief  Return the size of pre encrypted binary image header (`struct pre_enc_bin_header`). The initial header in the image contains magic, credentials (symmetric key) and few other parameters. This API could be useful for scenarios where the entire decrypted image length must be computed by the application including the image header.  
*
* @return
*    - uint16_t header_size
```

Reply:
Wording of the API should not really talk about any specific use-case, it should only talk about its implementation details and under what scenario it could be useful. My review comment is based on that. Please take care of formatting while making the change.